routes/routes.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+var router = express.Router();
+const controllerGet = require('../controller/get');
+const controllerPost = require('../controller/post');
+const controllerDelete = require('../controller/delete');
```

Comment:
Does it make sense to nest specific controllers under `../controller/posts`

Reply:
I didn't quite get this one. I think that I'll have only one POST endpoint, because for adding comments I'll be using PUT, or should it be POST?

Reply:
Sorry, i meant nesting post related controller files under `/controller/posts` directory.